pkg/operator/upgradebackupcontroller/upgradebackupcontroller.go

```diff
@@ -31,6 +31,8 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// NOTE: This controller DISABLED in 4.10+ and can be removed at any time.
```

Comment:
We should probably not run the tests anymore for this package and skip them instead?

https://github.com/openshift/cluster-etcd-operator/blob/480a387c0f5b3f03478bce1d44a2b83d7416a320/pkg/operator/upgradebackupcontroller/upgradebackupcontroller_test.go

Reply:
yeah makes sense